.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,61 @@
+name: Node CI
+on:
+  push:
+    branches:
+      - develop
+      - gh-pages
+  pull_request: {}
```

Comment:
This is triggered [whenever a PR changes](https://help.github.com/en/articles/events-that-trigger-workflows#webhook-events):

>Triggered when a pull request is assigned, unassigned, labeled, unlabeled, opened, edited, closed, reopened, synchronize, ready_for_review, locked, unlocked or when a pull request review is requested or removed. See below to learn how this event works with forked repositories.

Maybe just do `push` without `branches` limit?

Reply:
I did test if it got triggered on changes mentioned there (I assigned myself this PR) and nothing happened. Probably a documentation error.